edenai_apis/apis/deepgram/deepgram_api.py

```diff
@@ -177,3 +188,60 @@ def audio__speech_to_text_async__get_job_result(
             standardized_response=standardized_response,
             provider_job_id=public_provider_job_id,
         )
+
+    def audio__text_to_speech(
+        self,
+        language: str,
+        text: str,
+        option: str,
+        voice_id: str,
+        audio_format: str,
+        speaking_rate: int,
+        speaking_pitch: int,
+        speaking_volume: int,
+        sampling_rate: int,
+    ) -> ResponseType[TextToSpeechDataClass]:
+        _, model = voice_id.split("_")
+        base_url = f"https://api.deepgram.com/v1/speak?model={model}"
+        if audio_format:
+            base_url += f"&container={audio_format}"
+
+        if sampling_rate or sampling_rate != 0:
```

Comment:
I think there was a typo here, didn't you mean to do: 
```python
if sampling_rate and sampling_rate !=0:
```
because with the `or`, a **None** value would be accepted